src/components/App/Menu.tsx

```diff
@@ -51,7 +51,12 @@ export const Menu = (): JSX.Element => {
             </SheetClose>
 
             <SheetClose asChild>
-              <a href={`${import.meta.env.BASE_URL}/api/auth/signout`}>
+              <a href='' onClick={(event) => {
+                event.preventDefault()
+                signOut()
+                  .catch((error) => console.log(error))
```

Comment:
Shouldn't we do something when this happens? I understand it is not that important, but still?

Reply:
Yes, we really should do something. But I think that's a whole different task. We should show this to the user. I guess console.error() would be better but in the end it's the same result.

I'll change to console.error so we'll find it once we'll get to fixing it.